targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L496xG/TARGET_NUCLEO_L496ZG/PinNames.h

```diff
@@ -317,6 +317,14 @@ typedef enum {
     SYS_WKUP4 = PA_2,
     SYS_WKUP5 = PC_5,
 
+    /**** QSPI pins ****/
+    QSPI1_IO0 = PE_12,
+    QSPI1_IO1 = PB_0,
```

Comment:
There is no QSPI in NUCLEO_L496...

Reply:
@jeromecoutant so why QSPI pins are configured in https://github.com/ARMmbed/mbed-os/blob/3057c6c7223d6132a1cbfff35d90581a45b19bec/targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L496xG/TARGET_NUCLEO_L496ZG/PeripheralPins.c#L413-L460

and described on as routed out to external connector on NUCLEO_L496ZG scheme?

Reply:
STM32L496ZG chip have some QSPI pins.

NUCLEO board doesn't have QSPI embedded memory.
If a customer is connecting one, he can.